pulsar_spectra/catalogue_papers/Johnston_1992.yaml

```diff
@@ -2,11 +2,11 @@
  "J1012-5857": {
   "Frequency MHz": [
    640.0,
-   1577.0
```

Comment:
I think we were both wrong on this one as it says, "System Two consisted of a 2 x 64 x 5 MHz filterbank and the 320-MHz bandwidth was centred on 1520 MHz.". If you agree with the text please change these values to 1520.0

Reply:
Sure